src/utils/translations/gu.ts

```diff
@@ -0,0 +1,94 @@
+const gu = {
+  // Home Tab
+  Home: "ઘર",
+  "Period in": "માસિક ધારામાં",
+  "no info": "માહિતી વિના",
+  Period: "માસિક",
+  today: "આજે",
+  Delay: "વિલંબ",
+  Days: "(૦)[દિવસ];(૧)[દિવસ];(૨-૨)[દિવસ];(૫-૨૦)[દિવસ];(૨૧)[દિવસ];(૨૨-૨૪)[દિવસ];(૨૫-૩૦)[દિવસ];(૩૧)[દિવસ];(૩૨-૩૪)[દિવસ];(૩૫-૪૦)[દિવસ];",
```

Comment:
You are writing the same word in any case, so you just can use `(0-inf)` instead